### DISCUSSION TITLE
There is something wrong with the application the student pack

### DISCUSSION THREAD

**Discussion by LingYunNetWork** [community/community#20328] [GitHub Education]
Hello everyone.I am a student from China.And I am going to the senior high.Today,I tried to get my student pack.However,I found sth wrong:
Students in China Have NO ID cards with thier School’s name.So I uploaded my transcript.
My school and my name are on it:
![a39194587534da0e5de5d9422ea983a5fb6db05c](https://user-images.githubusercontent.com/108985539/178249147-77f9ea9c-ca77-4145-9c4e-fd303fe69858.jpeg)

However,When I tried to upload it.An error message appeared:
It said that The name of my school was not in the image
I still submitted, but as far as I know, applications with similar warnings tend to be post-processed.
I think this is caused by the image recognition not being able to recognize Chinese.
What should I do now?
If anyone could help me I would be very grateful.

(sorry for my bad English

2022/7/10 NEW:

I felt so bad because they TURNDOWN my application  : (

What can I do now???
I have many documents to prove that I am a student.However,none of the are in English...
Also,It is impossible for me to get documents in English in China!

**Comment by LingYunNetWork** [comment]
![image](https://user-images.githubusercontent.com/108985539/178249595-8e03c247-e8a4-46ec-a5b7-6a7308961123.png)

**Comment by davelsio** [comment]
This whole system is awful. GitHub, you need to fix the application process! Only on the first page (10 posts) of this discussion forum are **4** posts detailing problems with the system, and they are all going unanswered by support.

Sending a ticket to support does not seem to help either, as I mentioned in [another comment](https://github.com/orgs/github-community/discussions/19288#discussioncomment-3144987), because they get closed by a bot with a generic answer.

#19288
#20115
#17736

**Reply by LingYunNetWork** [reply]
So bad  :(
But I will try to send a ticket.Although it may get closed.
Thanks

**Reply by LingYunNetWork** [reply]
Oh shit
![image](https://user-images.githubusercontent.com/108985539/179971318-12738f57-36c4-40c7-ac00-5eb62d6d2d5a.png)
I tried to ask for help.But their employees' answers were perfunctory
